api/db/populate-users.js

```diff
@@ -132,7 +134,7 @@ const createUsers = async (multiplier, randomLocation, verbose) => {
       dateOfBirth: getRandomDate(
         new Date(1970, 0, 1),
         new Date().setFullYear(now.getFullYear() - 13)
-      ),
+      ).setUTCHours(0, 0, 0, 0),
```

Comment:
This should also be applied to the admin user, although there you can write it directly as `new Date(2000, 0, 1, 0, 0, 0, 0)`